lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -388,12 +388,16 @@ def _read_fq_record(self, source_obj_ref, source_obj_name,
                 l = f.readline()
             if not l:  # EOF
                 if i != 0:
+                    error_message_bindings = [shock_node, shock_filename]
+                    error_message = 'Reading FASTQ record failed - non-blank lines are not a multiple of four. '
+                    if source_obj_ref is not None and source_obj_name is not None:
+                        error_message += 'Workspace reads object {} ({}), '
+                        error_message_bindings.insert(0,source_obj_ref)
+                        error_message_bindings.insert(0,source_obj_name)
+                    error_message += 'Shock node {}, Shock filename {}'
                     raise ValueError(
-                        ('Reading FASTQ record failed - non-blank lines ' +
-                         'are not a multiple of four. Workspace reads ' +
-                         'object {} ({}), Shock node {}, Shock filename {}')
-                        .format(source_obj_name, source_obj_ref,
-                                shock_node, shock_filename))
+                        (error_message)
```

Comment:
You don't need the params around error_message here - they were there originally so the .format() call applied to the whole string rather than just the last line.